src/content/nft/index.md

```diff
@@ -0,0 +1,387 @@
+
+- Fractionalised NFTs can be traded on [DEXs](/defi/#dex) like Uniswap, not just [NFT marketplaces](/dapps?category=collectibles). That means more buyers and sellers.
+- An NFT's overall price can be defined by the price of its fractions.
+- You have more of an opportunity to own and profit from items you care about. It's harder to be priced out of owning NFTs.
+
+This is still experimental but you can learn more about fractional NFT ownership at the following exchanges:
+
+- [NIFTEX](https://landing.niftex.com/)
+- [NFTX](https://gallery.nftx.org/)
+
+In theory, this would unlock the possibility to do things like own a piece of a Picasso. You would become a shareholder in a Picasso NFT, meaning you would have a say in things like revenue sharing. It's very likely that one day soon owning a fraction of an NFT will enter you into a decentralised autonomous organisation (DAO) for managing that asset.
+
+These are Ethereum-powered organisations that allow strangers, like global shareholders of an asset, to coordinate securely without necessarily having to trust the other people. That's because not a single penny can be spent without group approval.
+
+As we mentioned, this is an emerging space. NFTs, DAOs, fractionalised tokens are all developing at different paces. But all their infrastructure exists and can work together easily because they all speak the same language: Ethereum. So watch this space.
+
+[More on DAOs](/dao/)
+
+<Divider />
+
+## Ethereum and NFTs {#ethereum-and-nfts}
+
+Ethereum makes it possible for NFTs to work for a number of reasons:
+
+- Transaction history and token metadata is publicly verifiable – it's simple to prove ownership history.
+- Once a transaction is confirmed, it's nearly impossible to manipulate that data to "steal" ownership.
+- Trading NFTs can happen peer-to-peer without needing platforms that can take large cuts as compensation.
+- All Ethereum products share the same "backend". Put another way, all Ethereum products can easily understand each other – this makes NFTs portable across products. You can buy an NFT on one product and sell it on another easily. As a creator you can list your NFTs on multiple products at the same time – every product will have the most up-to-date ownership information.
+- Ethereum never goes down, meaning your tokens will always be available to sell.
+
+## The environmental impact of NFTs {#environmental-impact-nfts}
+
+NFTs are growing in popularity which means they're also coming under increased scrutiny – especially over their carbon footprint.
+
+To clarify a few things:
+
+- By pressing the button to create or buy an NFT you are not directly releasing carbon into the atmosphere.
+- NFTs aren't directly increasing the carbon footprint of Ethereum.
```

Comment:
I feel like these two bullets are saying the same thing. I think this one is more to the point.